lighthouse-core/lib/dependency-graph/simulator/network-analyzer.js

```diff
@@ -185,6 +185,23 @@ class NetworkAnalyzer {
     });
   }
 
+  /**
+   * @param {LH.NetworkRequest[]} records
+   * @return {boolean}
+   */
+  static canTrustConnectionInformation(records) {
+    const connectionIdWasStarted = new Map();
```

Comment:
`connectionIdWasNew` or something like that, maybe? As stand more opposite to `connectionReused `